LogiusWebApp/src/main/java/org/verapdf/crawler/api/monitoring/CrawlJobStatus.java

```diff
@@ -0,0 +1,52 @@
+package org.verapdf.crawler.api.monitoring;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.verapdf.crawler.api.crawling.CrawlJob;
+
+import java.util.List;
+
+/**
+ * @author Maksim Bezrukov
+ */
+public class CrawlJobStatus {
+
+	@JsonProperty
+	private CrawlJob crawlJob;
+	@JsonProperty
+	private HeritrixCrawlJobStatus heritrixStatus;
+	@JsonProperty
+	private List<String> documentsQueue;
```

Comment:
Not sure we need the whole list... I guess just the count and the last processed document is enough (or the one which is in progress)